packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -34,8 +35,8 @@ const CdpIntegrationScript = ({
   route: { itemLanguage, name },
   pageEditing,
 }: CdpIntegrationProps): JSX.Element => {
-  const clientKey = process.env.BOXEVER_CLIENT_KEY
-  const targetUrl = process.env.BOXEVER_TARGET_URL
+  const clientKey = process.env.BOXEVER_CLIENT_KEY;
+  const targetUrl = process.env.BOXEVER_TARGET_URL;
```

Comment:
you can remove these variables and just use env vars at the bottom

Reply:
done